storybook/StorybookArgs.ts

```diff
@@ -0,0 +1,113 @@
+import { Args } from '@storybook/react';
+
+/*
+ * This is the type as defined here: https://storybook.js.org/docs/react/api/arg-types#argtypes
+ * The actual library exports only `interface Args { [name: string]: any; }`.
+ * The "any" does not allow your editor to show autocomplete or documentation.
+ *
+ * This has been reported before: https://github.com/storybookjs/storybook/issues/11916
+ * If the official types get updated then we can remove this whole file
+ * and import directly Args from '@storybook/react'
```

Comment:
Just wondering: Would you like to do a PR in storybook to export those types?

Reply:
Yeah I'll look into that. I am afraid there's some roadblock - why else would the storybook people add the type in the docs but not in the code? But it's worth a try at least.